ingesters/preprocessors/preprocessors.md

```diff
@@ -12,18 +12,9 @@ Each preprocessor will have the opportunity to modify the entries. The preproces
 
 ## Configuring Preprocessors
 
-Preprocessors can be used with the following ingesters:
+Preprocessors are supported on all packaged ingesters, one-off and unsupported ingesters may not support preprocessors.
 
-* Simple Relay
-* Kinesis
-* Kafka
-* Google Pub/Sub
-* Office 365
-* HTTP
-
-The other Gravwell ingesters will receive preprocessor support soon.
-
-Preprocessors are configured in the ingester's config file. Consider the following example for the Simple Relay ingester:
+Preprocessors are configured in the ingester's config file using the `preprocessor` configuration stanza.  Each preprocessor stanza must declare the preprocessor module in use via the `type` configuration parameter, followed by the preprocessors specific configuration parameters. Consider the following example for the Simple Relay ingester:
```

Comment:
We should style these as Preprocessor and Type (capitalized).

Also, "preprocessor's specific configuration" (should be possessive)